content/_releases/23/04.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Product Update - v23.04
+description: Details of the product update v23.04 from 26th of January 2023.
+layout: rel
+version: "23.04"
+section: Product Updates
+category: product-updates
+releaseDate: 2023-01-26
+redirect_from:
+  - /releases/23.04.html
+---
+
+## Improvements
+
+### Embedded Recipe Page: Recipe Credentials Edit
+
+We are introducing the ability to create embedded page for editing credentials for a single resipe's deployment. For this purpose we have created a new API endpoint `GET /v2/recipe-deployments/:recipe_deployment_id`. This endpoint requires same permissions and scopes as in `/v2/recipe/:recipe_id?include=recipe-deployments`.
+
+> Please visit [API Docs](({{site.data.tenant.apiDocsUri}}/v2#/recipe%20deployments)) for more details on this API endpoint.
+
+> For more informaition on Embedded recipe deployment please read this [article](/guides/embedded-recipe-deployment/).
+
+### Create/Edit Recipe API-Call
+
+We are introdusing `POST /v2/flows/export-to-recipe` API Call. With this call you can:
+
+* export a set of flows to a recipe.
+* update existing recipe with set of new flows.
+
+This new, more powerful endpoint, replaces the old one - `POST /v2/flows/${flowId}/export-to-recipe`. Please visit [API Docs](({{site.data.tenant.apiDocsUri}}/v2#/recipes/post_flows_export_to_recipe)) for more details.
+
+### Flow Queues improvements
+
+Before, the Flow Queues page displays the components inverted according to the flow graph (e.g if graph contain fork or content-based router) and don't display the step name, only the number.
+
+Now we render step name on Flow Queues page (like on Flow Designer) and render flow graph at Flow Queues, Runlog Execution, Thread pages in the same way as at Flow Designer page.
+
+{% include img.html max-width="60%" url="/assets/img/RN/2304/rn2304-flow-queues.png" title="Flow Queues" %}
+
+{% if site.data.tenant.name == "elastic.io" %}
+
```

Comment:
### Maximum frequency of flow scheduling changed to 10 minutes

As part of our work to make the platform more robust and tolerant of high loads, we have decreased the maximum frequency of scheduling a flow to start. This default setting ensures that flows will be started more precisely. Even if the flow schedule is defined in the settings page as running every 3 minutes, these flows will now run only every 10 minutes.
Customers wishing to trigger flows more frequently must change the flow to run in Real-time mode.